### ISSUE TITLE
no-unused-variable 

### ISSUE THREAD

**Issue by isergey** [palantir/tslint-react#31]
no-unused-variable warning highlighting **defaultProps** and **componentDidMount**. I think it is wrong, is not it?
```typescript
export default class App extends React.Component<any, ICounterState> {
  private static defaultProps = {
    title: "wefwef",
  };
  private unsubscribe: any;
  constructor(props: any) {
    super(props);
    this.state = store.getState();
  }
  public render() {
    return (
      <div>
        <button onClick={this.handlePlusClick}>+</button>
        <button onClick={this.handleMinusClick}>-</button>
        <div>{this.state.counter}</div>
      </div>
    );
  }
  private componentDidMount() {
    this.unsubscribe = store.subscribe(this.handleStoreUpdate);
  }
  private handleStoreUpdate = () => {
    this.setState(store.getState());
  }
  private handlePlusClick = () => {
    store.dispatch(increment());
  }
  private handleMinusClick = () => {
    store.dispatch(increment());
  }
}
```

**Comment by JKillian** [comment]
These should be `public` instead of `private` to get the error to go away.
